source/administration/changelog.md

```diff
@@ -259,9 +259,9 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)
 
-- **v5.31.2, release day TBD**
-  - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
-  - Fixing bugs related to replication lag for Enterprise Edition instances configured to use read replicas. [MM-31094](https://mattermost.atlassian.net/browse/MM-31094)
+- **v5.31.2, released 2021-03-29**
+  - Improved typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```

Comment:
Question I heard from JasonB - what does ``on busy servers`` mean? Lots of users? Lots of channels? Lots of messages? @lieut-data If we want to update this text, I can update it also for the v5.33 dot release section.

Reply:
I don't have numbers to make this concrete, but in short this problem will get worse as the number of concurrent, active users goes up.

Reply:
I updated the text to mention lots of active users.